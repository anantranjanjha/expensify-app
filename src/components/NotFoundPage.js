import React from "react";

const NotFoundPage = () => (
    <div>
        NotFoundPage Showing
    </div>
)

export default NotFoundPage;