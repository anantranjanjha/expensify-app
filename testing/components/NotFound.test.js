import React from "react";
import { shallow } from "enzyme";
import NotFoundPage from '../../src/components/NotFoundPage';

test('testing React Router Component', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
})