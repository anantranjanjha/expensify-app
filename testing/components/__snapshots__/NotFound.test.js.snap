// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing React Router Component 1`] = `
<div>
  NotFoundPage Showing
</div>
`;
